src/jit/hwintrinsiclistxarch.h

```diff
@@ -11,59 +11,69 @@
 // clang-format off
 
 #if FEATURE_HW_INTRINSICS
-//                  Intrinsic ID                   Function name                ISA   
+// **************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
+//                  Intrinsic ID                          Function name                            ISA        return type             parameter#            ival       SIMD size                                                                             instructions                                                                                       Category
+//                                                                                                        (UNDEF if multi types)   (-1 if multi numbers)                                    {TYP_BYTE,      TYP_UBYTE,     TYP_SHORT,     TYP_USHORT,    TYP_INT,       TYP_UINT,      TYP_LONG,      TYP_ULONG,     TYP_FLOAT,     TYP_DOUBLE}
```

Comment:
are TYP_BYTE and TYP_UBYTE instructions ever different (other than the case where one is valid and one is invalid)?

Reply:
(question also applies to the other signed vs unsigned types)

Reply:
Yes, please see multiply in this PR.